docs/guides/server/vault.adoc

```diff
@@ -46,6 +56,24 @@ sso__realm_ldap__credential
 ----
 Note the doubled underscores between __sso__ and __realm__ and also between __ldap__ and __credential__.
 
+== Configuring the Java KeyStore-based vault
+
+In order to use the Java KeyStore-based vault, you need to create a KeyStore file first. You can use the following command for doing so:
+[source, bash]
+----
+keytool -importpass -alias test_alias -keystore keystore.p12 -storepass keystorepassword
+----
+and then enter a value you want to store in the vault.
+This by default results to storing the value in a form of generic PBEKey (password based encryption) within SecretKeyEntry using `PBEWithMD5AndDES` as the algorithm.
```

Comment:
Before having this change in KC 22. Would be possible to update/replace the algorithm provided by Keycloak Vault?

MD5 and DES combination cannot easily be broken but brute-forced, considering that we are aiming to enhance security, I'd suggest replacing it by PBEWithHmacSHA256AndAES_128, or PBEWithHmacSHA256AndAES_256 if possible.

@JoshuaMulliken could you please review? The proposal is based on https://github.com/keycloak/keycloak/discussions/19281

Reply:
I agree and we should 100% utilize the aes256 variant as it will set us up for success when the Feds post quantum encryption requirements come into affect

Reply:
@vmuzikar @Pepo48 can we change the instructions to something like the snipped below?

```
keytool -importpass -alias test_alias -keystore keystore.p12 -keyalg PBEWithHmacSHA256AndAES_256 -storepass keystorepassword
```

Or it does not matter what we do, the keystore will always use the default?

Reply:
@abstractj I actually had the very same idea while testing it, but based on my experience this does not change Secret Key Entry algorithm. Here's a quick proof:
![image](https://user-images.githubusercontent.com/12138171/231476154-16a9d72d-9485-40cc-af4e-115238552bce.png)